controller/api/destination/endpoint_translator.go

```diff
@@ -28,6 +28,8 @@ const (
 	envControlListenAddr = "LINKERD2_PROXY_CONTROL_LISTEN_ADDR"
 
 	updateQueueCapacity = 100
+
+	defaultProxyInboundPort = 4143
```

Comment:
Is it worth deriving that from the values.yml? Again, not too sure. If we are using that only for external workloads, what is in the values.yml does not have much relationship to what is configured on the VM.

Reply:
Well, I guess if you do alter the global value in values.yaml, you'd want that default to be honored right? On the other hand, this is an objective improvement of the current behavior, so it's not necessarily a blocker.

Reply:
Frankly, I am not really sure. Two reasons: 

- we will use this value when addressing external workloads - their inbound port is not associated with what Linkerd is configured with in the cluster
- seems to me that same argument can be made for situations when we do discovery for remote clusters. If we do discovery for an endpoint that lives in a remote cluster, we would rely on the value that Linkerd has been installed with in the local cluster, not the remote one.

I am happy to do a follow up and make that configurable if these two points are not a problem. For now, will go ahead and merge that as is since you mention this is not a blocker.